web_widget_text_markdown/tests/test_ui.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Alexandre Díaz <dev@redneboa.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import HttpCase
+
+
+class TestUi(HttpCase):
+
+    post_install = True
+    at_install = False
+
+    def test_ui_web_field_text_markdown(self):
+        self.phantom_js(
+            "/web/tests?module=fields&failfast",
```

Comment:
This is weird. Maybe you need this?:

```suggestion
            "/web/tests?module=web_widget_text_markdown&failfast",
```